### PR TITLE
Avatar badge

### PR DESCRIPTION
**Pull request by techieshubham30**

### CODE REVIEW

File: Components/Badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+.badge-container{
+    display: flex;
+    position: relative;
+    width: fit-content;
+    font-size: 2rem;
+}
+
+.status-badge{
+    border-radius: 50%;
+    border: 2px solid #fff;
```
Comment (KoushikKintali) [inline Components/Badge/badge.css:11]: declare colors as variable & use it here

**Review by KoushikKintali — APPROVED** [review]
just  a small comment

**Review by himadri2110 — APPROVED** [review]
Good to go!

File: Components/Badge/badge.css (hunk 1 above)
Comment (techieshubham30) [reply Components/Badge/badge.css:11]: Sure, will do that thanks!

**Pull request merged**